pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -1911,8 +1911,10 @@ var (
 	DefaultInPlaceWorkerMaxSurge = intstr.FromInt32(0)
 	// DefaultWorkerMaxUnavailable is the default value for Worker MaxUnavailable.
 	DefaultWorkerMaxUnavailable = intstr.FromInt32(0)
-	// DefaultInPlaceWorkerMaxUnavailable is the default value for In-Place Worker MaxUnavailable.
-	DefaultInPlaceWorkerMaxUnavailable = intstr.FromInt32(1)
+	// DefaultAutoInPlaceWorkerMaxUnavailable is the default value for AutoInPlaceUpdate Worker MaxUnavailable.
+	DefaultAutoInPlaceWorkerMaxUnavailable = intstr.FromInt32(1)
+	// DefaultManualInPlaceWorkerMaxUnavailable is the default value for ManualInPlaceUpdate Worker MaxUnavailable.
```

Comment:
Maybe add a comment about why this is 0, because `MaxUnavailable` is not respected for manual in-place workers.